pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "sail-on-client"
+version = "0.15.0+29.7963669"
+description = "Client and Protocols for DARPA sail-on"
+authors = ["Your Name <you@example.com>"]
```

Comment:
You may want to update this or delete it.

Reply:
Hmm good catch I'll update it

Reply:
@waxlamp PTAL